sdk/go/kubernetes/yaml/yaml.go

```diff
@@ -356,6 +356,15 @@ func parseYamlObject(ctx *pulumi.Context, obj map[string]interface{}, transforma
 
 	key := fmt.Sprintf("%s::%s", fullKind, metaName)
 
+	if fullKind == "v1/Secret" {
```

Comment:
For the Go SDK, I had to make this change in two places since the YAML support bypasses the `NewSecret` constructor in favor of manually registering the resource with `UntypedArgs`.

Reply:
Just confirming that the same is not true for other sdks' yaml support (bypassing resource construction in favor of manually registering resources)? Would also be good to cut tickets to track such deviations across SDKs.

Reply:
This is true for both .NET and Go due to the stricter typing in those SDKs.